documentation/src/docs/asciidoc/docinfos/index-docinfo.html

```diff
@@ -0,0 +1 @@
+<link rel="icon" type="image/png" href="{{ site.baseurl }}/img/junit5-logo.png">
```

Comment:
`{{ site.baseurl }}` will not be replaced by Jekyll because the resulting index.html does not have YaML frontmatter. Please add junit5-logo.png to the images folder and reference it.